src/utils/serverRunner.js

```diff
@@ -23,7 +24,14 @@ function handleProcessErrors(server) {
       Logger.close()
     ])
     .catch(() => {}) // ignore errors thrown here
-    .then(() => process.exit(-1));
+    .then(() => {
+      if (newrelic) {
+        newrelic.addCustomParameter('crash', 'true');
```

Comment:
Please put this code in the ``utils/newrelic`` util in a ``crashClose`` method (or something like that), which returns a promise and add that to the ``Promise.all`` above